docs/standards/javascript_standards.md

```diff
@@ -2,13 +2,27 @@
 
 Use vanilla JavaScript, don't use extensions to the language such as TypeScript
 
-### Use [Standard JS](https://standardjs.com/) to lint your code
+### Use [neostandard](https://github.com/neostandard/neostandard) to lint your code
+
+neostandard is an opinioned JavaScript linter that enforces a consistent coding style.
 
 - A consistent approach to code layout, spacing and formatting makes it easier to switch between projects.
 
-By [adding it as a dev dependency](https://standardjs.com/index.html#install) it can be easily used in the terminal,
-[Webstorm](https://blog.jetbrains.com/webstorm/2017/04/using-javascript-standard-style/),
-and [Visual Studio](https://marketplace.visualstudio.com/items?itemName=chenxsan.vscode-standardjs).
+- By adding it as a dev dependency it can be easily used in the terminal and CI pipelines.
+
+- The choice of an opinionated linter is deliberate as it avoids teams needing to spend time debating the merits of different styles.
+
+- Teams should not extend or modify the neostandard ruleset.
```

Comment:
Is it worth being a bit more explicit and saying **do not** use the `noStyle` option (i.e. making sure you are using neo's styling rules)? I think we discussed this as a possibility to keep it as opinionated as possible.

`noStyle option deactivates style rules (used to require something like eslint-config-prettier)`

Reply:
Added as a separate point as don't want to cloud the simple message of don't change it